dist/api.classic.js

```diff
@@ -111,7 +113,11 @@
             executeFn(disconnectFn)();
         }.bind(this), raiseError(errorFn, "connect error"));
     };
+    Device.prototype.isConnected = function() {
+        return this.connected;
```

Comment:
In fact, if it is initiated, you don't need a getter function and can just use the device property `device.connected`